packages/@webex/plugin-cc/src/services/AgentService.ts

```diff
@@ -56,4 +64,28 @@ export default class AgentService {
       return Promise.reject(error);
     }
   }
+
+  public async getBuddyAgents(options: BuddyAgents): Promise<BuddyAgentsSuccess> {
```

Comment:
Return type for all the methods should be generic not just associated with success